oemoflex/tools/plots.py

```diff
@@ -286,24 +286,20 @@ def assign_stackgroup(key, values):
 
 def plot_dispatch_plotly(
     df,
-    bus_name,
-    demand_name="demand",
+    df_demand,
     colors_odict=colors_odict,
     unit="W",
```

Comment:
Maybe it is better if the user has to pass the unit explicitly in any case, which means we would abandon the default "W".